### ISSUE TITLE
REBUILD OF NOCLIP: You Can (Not) Level Design

### ISSUE THREAD

**Issue by azazellus** [VDP-noclip/noclip#202]
We need to talk.

I discussed with @TizioMaurizio and reached the conclusion that the actual level design makes no sense. The environment has no context and the noclip mode is not interesting since it has not a narrative purpose.

But we had an idea: what if _time was convoluted?_ What if the noclip mode was just a way we could see the past? What if invisible and intangible objects were simply the effect of a time glitch?

We could steal from Halo (again) the concept of ancient but technological ruins and use it to build the puzzles. This way puzzles and environment would be more harmonic and appealing, giving the player a narrative reason to play (= discovery).

Since in the feedback someone suggested to make puzzles more skill based than memory based, we could also change the noclip mechanic, making noclipping a limited consumable power that you can activate wherever you want.

This massive maneuver at three weeks from the deadline means a few things:
- we need more than one level designer;
- we need LOTS of ideas;
- we need rock solid planning.

What do you think?
ARE YOU IN????
ARE WE GONNA MAKE THE BEST GAME EVER RELEASED????
ARE WE GONNA MAKE HISTORY?????

**Comment by TizioMaurizio** [comment]
I agree on the fact that the game developement should be level driven, otherwise we are just elaborating an idea that will not work in the real implementation, hence the magnitude and need of a noclip mechanics redesign should be evaluated and measured on actual playable and testable levels and drafts.
Tools like probuilder can and should be used to suggest level ideas and increase the testing material.
Since we are developing a game we should be the first ones to play around with it if we want the final result to be fun.

**Comment by stefanofossati** [comment]
Honestly I haven't understood what do you mean, when we will speak maybe it will be more clear.

I agree with the idea of creating a rigid planning in order to do everithing, so that everyone can organize himself

**Comment by TizioMaurizio** [comment]
Looking at the labels of this issue we can tell that the Christmas tree for this year is ready
